diff -crN ./nginx-1.7.6.orig/src/http/ngx_http_core_module.h ./nginx-1.7.6/src/http/ngx_http_core_module.h
*** ./nginx-1.7.6.orig/src/http/ngx_http_core_module.h	2014-09-30 22:20:34.000000000 +0900
--- ./nginx-1.7.6/src/http/ngx_http_core_module.h	2014-10-17 18:47:45.000000000 +0900
***************
*** 578,589 ****
          r->headers_out.location = NULL;                                       \
      }
  
! #define ngx_http_clear_etag(r)                                                \
!                                                                               \
!     if (r->headers_out.etag) {                                                \
!         r->headers_out.etag->hash = 0;                                        \
!         r->headers_out.etag = NULL;                                           \
!     }
  
  
  #endif /* _NGX_HTTP_CORE_H_INCLUDED_ */
--- 578,584 ----
          r->headers_out.location = NULL;                                       \
      }
  
! #define ngx_http_clear_etag(r)
  
  
  #endif /* _NGX_HTTP_CORE_H_INCLUDED_ */
